docs/source/lesson/prg/python_intro.rst

```diff
@@ -1013,12 +868,119 @@ we can run it like so::
 
 Let break this down a bit.
 
-#. We are importing the ``print_function`` and ``division`` modules from Python 3 like we did earlier in this tutorial. It's a good idea to always include these in your programs.
-#. We've defined the ``check_triangle`` function which has three parameters. It returns ``True`` if all six expressions defined inside are true, and ``False`` otherwise.
-#. We've used the backslash symbol ``\`` to format are code nicely. The backslash simply indicates that the current line is being continued on the next line.
-#. When we run the program, we do the check ``if __name__ == '__main__'``. ``__name__`` is an internal Python variable that allows us to tell whether the current file is being run from the command line (value ``__name__``), or is being imported by a module (the value will be the name of the module). Thus, with this statement we're just making sure the program is being run by the command line.
-#. We are using the ``docopt`` module to handle command line arguments. The advantage of using this module is that it generates a usage help statement for the program and enforces command line arguments automatically. All of this is done by parsing the docstring at the top of the file.
-#. In the ``print`` function, we are using `Python's string formatting capabilities <https://docs.python.org/2/library/string.html#format-string-syntax>`_ to insert values into the string we are displaying.
+#. We are importing the ``print_function`` and ``division`` modules
+   from Python 3 like we did earlier in this tutorial. It's a good
+   idea to always include these in your programs.
```

Comment:
Why? You may want to expand a bit on the earlier brief description of these features.